<launch>
    <arg name="rate" default="10" />
    <arg name="max_time_deviation" default="0.1" />
    <arg name="base_link_frame" default="ugr/car_base_link" />

    <node pkg="ublox_gps" type="estimate_heading.py" name="estimate_heading" output="screen">
        <param name="rate" value="$(arg rate)" />
        <param name="max_time_deviation" value="$(arg max_time_deviation)" />
        <param name="base_link_frame" value="$(arg base_link_frame)" />

        <remap from="/input/gps0" to="/ugr/car/sensors/gps/ublox/left"/>
        <remap from="/input/gps1" to="/ugr/car/sensors/gps/ublox/right"/>

        <remap from="/output/heading" to="/ugr/car/sensors/heading"/>
    </node>
</launch>